### ISSUE TITLE
Function with multiple arguments as attribute value

### ISSUE THREAD

**Issue by vursen** [slime-lang/slime#124]
Hi! I got a problem when i use multiple argument's function:

```slime
script src=static_path(@conn, "/js/zepto.min.js")
```

```
== Compilation error on file web/views/layout_view.ex ==
** (TokenMissingError) web/templates/layout/app.html.slime:1: missing terminator: ) (for "(" starting at line 1)
    (eex) lib/eex/compiler.ex:36: EEx.Compiler.generate_buffer/4
    (phoenix) lib/phoenix/template.ex:378: Phoenix.Template.compile/2
    (phoenix) lib/phoenix/template.ex:186: Phoenix.Template."-MACRO-__before_compile__/2-fun-0-"/3
    (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
    (phoenix) expanding macro: Phoenix.Template.__before_compile__/1
    web/views/layout_view.ex:1: App.LayoutView (module)
    (elixir) lib/kernel/parallel_compiler.ex:117: anonymous fn/4 in Kernel.ParallelCompiler.spawn_compilers/1
```
As a temporary solution I use string interpolation:

```script src="#{static_path(@conn, "/js/main.bundle.js")}"```

**Comment by Rakoth** [comment]
Hi! This is a known issue, as another work-around you can remove space after comma between arguments. Also there is PR #113 with a fix.

**Comment by Rakoth** [comment]
closed by #113
